src/orange/orangeinp/IntersectRegion.hh

```diff
@@ -13,6 +13,8 @@
 #include "corecel/math/Turn.hh"
 #include "orange/OrangeTypes.hh"
 
+#include "../surf/detail/InvoluteSolver.hh"
```

Comment:
Let's not include the involute solver detail in this unrelated class. Let's add a new enum to `OrangeTypes`:
```c++
//---------------------------------------------------------------------------//
/*!
 * Chirality of a twirly object (currently only Involute).
 */
enum class Chirality : bool
{
    left, //!< Sinistral, spiraling counterclockwise
    right, //!< Dextral, spiraling clockwise
};
```

You can update the existing typedefs to reduce the amount of code you have to change:
```c++
    using Sign = Chirality;
    using ccw = Chirality::left;
    using cw = Chirality::right;
```